### ISSUE TITLE
Link to ShuffleNetV2

### ISSUE THREAD

**Issue by lxtGH** [implus/PytorchInsight#13]
How to download such pretrained model?
